packages/auth/src/core/util/handler.ts

```diff
@@ -40,6 +40,13 @@ const WIDGET_PATH = '__/auth/handler';
  */
 const EMULATOR_WIDGET_PATH = 'emulator/auth/handler';
 
+/**
+ * Fragment name for the App Check token that gets passed to the widget
+ *
+ * @internal
+ */
+const FIREBASE_APP_CHECK_FRAGMENT_ID = 'fac';
```

Comment:
This probably doesn't need to be encoded down below, but to be safe and clear about it I think it's fine to keep `encodeURIComponent`. But I would move that call to up here instead of doing it in the function.

Reply:
Sounds good, updated